test-two/src/index.js

```diff
@@ -3,24 +3,49 @@ import { getUsersForOrganisation } from "./services/github";
 
 const baseUrl = "https://api.github.com/";
 const organisationId = "google";
-const gitHubApiAuthToken = ""; // add your GitHub API OAuth key here to increase request limit
+const gitHubApiAuthToken = "cf33f5689e7e8399aadb5cbaae8ef51bc2be7c66"; // add your GitHub API OAuth key here to increase request limit
+
+/*
+  I was unable to get my program to work in the desired way but I managed a very
+  low quality version.
+
+  My initial plan was to iterate through the program, everytime the response headers
+  had a link with 'rel=next' I would loop.
+  Or I would take the page number from 'rel=last' and loop that many times.
+  But I was unable to access the data in the header as I think it is a nested
+  java object and it kept returning "undefined".
+
+  So I manually found the last page which ended up being 72, so I just iterated
+  that many times.
+
+*/
+
 
 export const go = async () => {
+  var pagenum;
```

Comment:
No issue really, but I prefer variable names that are not abbreviated so it's always clear what the variable is for and code is more readable. Uglify will reduce production bundles.

Reply:
Definitely noted :)